src/patient/PatientRecord.jsx

```diff
@@ -814,19 +813,19 @@ class PatientRecord {
 
     getBreastCancerGeneticAnalysisPanelsChronologicalOrder() {
         let panels = this.getEntriesOfType(FluxBreastCancerGeneticAnalysisPanel);
-        panels.sort(this._clinicallyRelevantTimeTimeSorter);
+        panels.sort(this._relevantTimeTimeSorter);
         return panels;
     }
 
     getGastrointestinalStromalTumorCancerGeneticAnalysisPanelsChronologicalOrder() {
         let panels = this.getEntriesOfType(FluxGastrointestinalStromalTumorCancerGeneticAnalysisPanel);
-        panels.sort(this._clinicallyRelevantTimeTimeSorter);
+        panels.sort(this._relevantTimeTimeSorter);
         return panels;
     }
 
     getPathologyReportsChronologicalOrder() {
         let reports = this.getPathologyReports();
-        //reports.sort(this._clinicallyRelevantTimeTimeSorter);
+        //reports.sort(this._relevantTimeTimeSorter);
```

Comment:
Should we uncomment this? If not, do we know what the justification was for commenting out in the first place? I know you weren't the originator of this, but thought I'd ask.

Reply:
I just removed this line.  We have a JIRA to support multiple pathology reports.

Reply:
Sounds good !